install.sh

```diff
@@ -994,11 +994,21 @@ case "${SHELL}" in
     ;;
 esac
 
+# show different instructions if the user already has Homebrew in their PATH
+if grep -qs "eval \"\$(${HOMEBREW_PREFIX}/bin/brew shellenv)\"" "${shell_profile}"
+then
+cat <<EOS
+- Homebrew was already found in your PATH, run this command to activate it in your current shell:
```

Comment:
Should we check if it's actually already been run?

Reply:
I think that's an orthogonal concern, based on the various permutations of "I've added `shellenv` to my profile but my PATH's still wrong". If the problem to be solved is "reliably add `shellenv` to user's profile", methinks the necessary and sufficient test is "has it already been added?"

Reply:
Perhaps that might be the better approach for this. If Homebrew was already in their current shell environment, I would assume the user would probably be aware of it.